learning_journal/learning_journal/views.py

```diff
@@ -0,0 +1,95 @@
+from pyramid.response import Response
+from pyramid.view import view_config
+import transaction
+import datetime
+from jinja2 import Markup
+import markdown
+from sqlalchemy import update
+from .forms import EntryCreateForm, EntryUpdateForm
+from pyramid.httpexceptions import HTTPFound
+
+
+from sqlalchemy.exc import DBAPIError, ResourceClosedError, IntegrityError
+
+from .models import (
+    DBSession,
+    Entry,
+)
+
+
+# @view_config(route_name='list', renderer='string')
+@view_config(route_name='list', renderer='templates/pretty.jinja2')
+def list_view(request):
+    display = DBSession().query(Entry.metadata.tables['entries']).all()[::-1][:10]
+    return {"content": display, "header": "entries.order_by(latest)[:10]"}
+
+
+@view_config(route_name='detail', renderer='templates/detail.jinja2')
+def detail_view(request):
+    id_ = request.matchdict.get('entry_id')
+    display = DBSession().query(Entry.metadata.tables['entries']).filter_by(id=id_).one()
+    md = markdown.Markdown(safe_mode='replace', html_replacement_text='--RAW HTML NOT ALLOWED--')
+    message = md.convert(display[2])
+
+    return {"message": message, "header": display[0], "title": display[1], "time": display[3]}
```

Comment:
This isn't a particularly model-centered way of working.  You're losing the fact that you can use your models to provide functionality.  Use your objects